assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	height: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Space = styled.div`
+	flex: 0 1 ${ ( { isNext } ) => isNext ? 60 : 115 }px;
+`;
+
+const NavButton = styled.button`
```

Comment:
Wouldn't it be easier to just use SVGs here? See related question at https://github.com/ampproject/amp-wp/pull/4106#discussion_r366668828

IMHO that would make it much easier to grasp than using CSS for it, and is also easier to reuse, for example in the carousel.

Reply:
Hmm, sure, will look into it.

Reply:
Will create new buttons under `component/button` directory.

Reply:
Reworked the buttons, mind checking again?